tests/containers/podman_bci_systemd.pm

```diff
@@ -18,8 +18,10 @@ sub run {
     my $podman = $self->containers_factory('podman');
     $self->{podman} = $podman;
 
+    record_info("Podman version", script_output("podman version"));
+
     record_info('Test', 'Launch a container with systemd');
-    assert_script_run("podman run -d -p 80:80 --health-cmd='curl http://localhost' --name nginx registry.suse.com/bci/bci-init:latest");
+    assert_script_run("podman run -d -p 80:80 --name nginx registry.suse.com/bci/bci-init:latest");
```

Comment:
How podman will know that the container is healthy? =/

Reply:
Podman doesn't need to know that in this case, but I can keep it